### PR TITLE
Add "Copy to Clipboard" context menu option

### LINKED ISSUE #110: Add a "Copy to Clipboard" context menu to the Bazel Workspace Tree view
It would be useful to provide a copy to clipboard option in the Bazel workspace Tree view. 
This would copy the target name so it can easily be added to the list of dependencies in a BUILD file.

### PR DESCRIPTION
**Pull request by VanBronckhorst**
Fixed #110.

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     },
     "icon": "media/bazel-logo.png",
     "engines": {
-        "vscode": "^1.27.0"
+        "vscode": "^1.30.0"
```
Comment (allevato) [inline package.json]: Is this change necessary for any of the functionality in this PR? If not, please revert it and also revert the changes to package-lock.json.

File: package.json [hunk 2]
```diff
@@ -81,6 +81,11 @@
                 "category": "Bazel",
                 "command": "bazel.showExtensionLog",
                 "title": "Show Extension Log"
+            },
+            {
+                "category": "Bazel",
+                "command": "bazel.copyTargetToClipboard",
+                "title": "Copy to Clipboard"
```
Comment (allevato) [inline package.json]: Let's call this "Copy Label to Clipboard" to make it clear what's being copied (the [label](https://docs.bazel.build/versions/master/build-ref.html#labels) as opposed to, say, file contents of some sort).

File: package.json (hunk 1 above)
Comment (allevato) [reply package.json]: A colleague pointed out that 1.30 is when the clipboard APIs were introduced. I've pushed a separate change to update the version requirement (#114); can you revert these package(-lock).json changes and rebase so this will be a standalone change?

Comment (VanBronckhorst) [reply package.json]: Yeah, that's why I changed the min-version, thanks for your change.

File: package.json (hunk 2 above)
Comment (VanBronckhorst) [reply package.json]: Ah thanks, I did not know the //path was called label.

**Comment by VanBronckhorst** [comment]
BTW this solves #110 , not #111  as per first comment
